src/ontology/mappings/import-corrections.sssom.tsv

```diff
@@ -0,0 +1,50 @@
+#curie_map:
+#  BFO: "http://purl.obolibrary.org/obo/BFO_"
+#  EMAPA: "http://purl.obolibrary.org/obo/emapa#"
+#  IAO: "http://purl.obolibrary.org/obo/IAO_"
+#  MA: "http://purl.obolibrary.org/obo/ma#"
+#  MmusDv: "http://purl.obolibrary.org/obo/MmusDv_"
+#  RO: "http://purl.obolibrary.org/obo/RO_"
+#  TS: "http://purl.obolibrary.org/obo/TS_"
+#  XAO: "http://purl.obolibrary.org/obo/xao#"
+#mapping_set_id: "http://purl.obolibrary.org/obo/uberon/import-corrections.sssom.tsv"
+#mapping_set_description: "Terms and properties to replace in imported ontologies."
+#license: "http://creativecommons.org/licenses/by/3.0/"
```

Comment:
My assumption is that this is "our" work, so I would suggest to use whenever possible CC-0. CC-BY 3 is a convoluted license, even BY-4 is better.

Reply:
I simply used the license of Uberon itself, which happens to be CC-BY 3.0:

```
property_value: dcterms-license http://creativecommons.org/licenses/by/3.0/
```

I have no strong opinion on that particular license versus CC-0, CC-BY-4.0, or even the WTFPL, but I don’t think it is a good idea to have different licenses for the different parts of a same project. (Except maybe for documentation, which may be under a different license than actual code.)